src/customStyle.ts

```diff
@@ -56,29 +58,45 @@ export function getCustomStyleByName(name: string): Style {
   return style;
 }
 
+export function setView(csview) {
+  _view = csview;
+}
+
 // store styles in cache
-export function setStyles(style: Style[]) {
+export function setStyles(style) {
   customStyles = style;
-}
+  let documentType;
+  if (style && Array.isArray(style)) {
+    documentType =
+      style.length > 0 && style[0].docType ? style[0].docType : null;
+  }
 
-export function setHidenumberingFlag(hideNumberingFlag: boolean) {
+  if (docType !== documentType) {
+    hasdocTypechanged = true;
+    docType = documentType;
```

Comment:
If `docType = documentType` regardless, why not just set `hasdocTypechanged = docType !== documentType`?